src/game-of-life/code-size.md

```diff
@@ -87,19 +87,19 @@ wasm-opt -O3 -o output.wasm input.wasm
 ### How small do these build configurations get our Game of Life `.wasm` binary?
 
 With the default release build configuration (without debug symbols), our
-WebAssembly binary is 240,605 bytes:
+WebAssembly binary is 29,410 bytes:
```

Comment:
We got it that small? Nice!

Reply:
I think this is mostly lld's --gc-sections removing unused panicking message fragments, since last time this was written, we didn't have that yet.